api/api/events/models.py

```diff
@@ -0,0 +1,17 @@
+from accounts.models import User
+from clubs.models import Club
+from django.db import models
+
+
+class Events(models.Model):
+    name = models.CharField(max_length=100)
+    description = models.TextField()
+    starts_at = models.DateField()
+    ends_at = models.DateField()
+    registration_required = models.BooleanField(default=False)
+    created_by = models.OneToOneField(User, null=True, on_delete=models.SET_NULL)
```

Comment:
`created_by` should be a `ForeignKeyField`.

Add `related_name` to all the `ForeignKeys`, `OneToOne` and `OneToMany` relationships.

Reply:
for `created_by` related name `events_created` works ?